.travis.yml

```diff
@@ -1,15 +1,33 @@
 language: python
-python: 3.4
+python:
+- 3.4
+  env:
+  - TOX_ENV=py34-cdh
+  - TOX_ENV=py34-hdp
+- 3.5
+  env:
+  - TOX_ENV=py35-cdh
+  - TOX_ENV=py35-hdp
+- 3.6
+  env:
+  - TOX_ENV=py36-cdh
+  - TOX_ENV=py36-hdp
+- 3.8
+  env:
+  - TOX_ENV=py38-cdh
+  - TOX_ENV=py38-hdp
+jobs:
+  allow_failures:
+    python:
+    - "3.4"
```

Comment:
Why continue to run Py34?
* https://devguide.python.org/#status-of-python-branches
* https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
I added back 3.4 after noticing 3.4 was still present in tox.ini and wondering if others had a use for it.